cdk/csharp/cdk.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app": "dotnet run --project src/src/Cdk/Cdk.csproj",
+  "watch": {
+    "include": [
+      "**"
+    ],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "src/*/obj",
+      "src/*/bin",
+      "src/*.sln",
+      "src/*/GlobalSuppressions.cs",
+      "src/*/*.csproj"
+    ]
+  },
+  "context": {
```

Comment:
Similar comment here: do we need all of these for this CDK project? Is this a common default or just a copy-n-past from another project?